src/agent/onefuzz-agent/src/local/common.rs

```diff
@@ -197,7 +215,7 @@ pub fn get_synced_dir(
 // fuzzing tasks from generating random task id to using UUID::nil(). This
 // enables making the one-shot crash report generation, which isn't really a task,
 // consistent across multiple runs.
-pub fn build_common_config(args: &ArgMatches<'_>, generate_task_id: bool) -> Result<CommonConfig> {
+pub fn build_local_context(args: &ArgMatches<'_>, generate_task_id: bool) -> Result<LocalContext> {
```

Comment:
Having a single context, does this mean that each task in the job shares the task_id?

Reply:
we manually generate a new task_id every time we pass the common_config to build a task specific config you can see it in libfuzzer.rs